<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1139</width>
    <height>686</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1139</width>
    <height>686</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1139</width>
    <height>686</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Openify</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="filesList">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>320</y>
      <width>421</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="currentList">
    <property name="geometry">
     <rect>
      <x>885</x>
      <y>30</y>
      <width>241</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="foldersList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>421</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Folders</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>580</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>635</y>
      <width>1041</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="timeStatus">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>637</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>--:--:--</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mediaTitle">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>22</y>
      <width>851</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18pt;</string>
    </property>
    <property name="text">
     <string>Nothing playing</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>290</y>
      <width>861</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>16</width>
      <height>571</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>886</x>
      <y>10</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Queue</string>
    </property>
   </widget>
   <widget class="QSlider" name="volume">
    <property name="geometry">
     <rect>
      <x>999</x>
      <y>597</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>411</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Album</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="album">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>Artist</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="artist">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Album's artist</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="albumArtist">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Composer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="composer">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>Year</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="year">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Genre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="genre">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>Comment</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="comment">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>27</height>
          </size>
         </property>
         <property name="text">
          <string>Codec</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="codec">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>Filename</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="filename">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
